plasma_framework/contracts/src/utils/TxPosLib.sol

```diff
@@ -0,0 +1,39 @@
+pragma solidity ^0.5.0;
+
+/**
+@dev transaction position = (blockNumber * BLOCK_OFFSET_FOR_TX_POS + txIndex).
+ */
+library TxPosLib {
+    struct TxPos {
+        uint256 value;
+    }
+
+    // Check the OFFSET values in UtxoPosLib, should be division of the values there.
+    uint256 constant internal BLOCK_OFFSET_FOR_TX_POS = 1000000000 / 10000;
+
+    /**
+     * @notice Given a TX position, returns the block number.
+     * @param _txPos position of transaction.
+     * @return The output's block number.
+     */
+    function blockNum(TxPos memory _txPos)
```

Comment:
haha, this is a fun story. So at midpoint of my refactoring `_transactionIncluded()` I think passing txPos rather than utxoPos makes more sense, as it is about `transaction` included or not. But than it is just a two line function and `Merkle.checkMembership()` is sort of good enough naming anyway, I abandoned the function.

Anyway, I checked the `Rootchain.sol`, there are several functions using `txPos` in in-flight exit so I think it makes sense to keep this since it is written with test already.